week2/assignment_2/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>TodoList</title>
+</head>
+<body>
+  <main>
+    <div class="btn_wrapper">
+      <button class="show-all">전체</button>
+      <button class="show-completed">완료됨</button>
+      <button class="show-incomplete">미완료</button>
+      <select name="show-priority">
+        <option selected disabled hidden>중요도</option>
+        <option>1</option>
+        <option>2</option>
+        <option>3</option>
+      </select>
+    </div>
+    <div class="todo-box">
+      <div class="input-box">
+        <input class="todo-value" placeholder="할 일을 입력하세요 !">
+        <select name="choose-priority">
+          <option value="" selected disabled hidden>중요도 선택</option>
+          <option value="1">1</option>
+          <option value="2">2</option>
+          <option value="3">3</option>
+        </select>
+        <button class="todo-add-btn">추가</button>
+      </div>
+      <div class="todo-actions-box">
+        <button class="todo-delete">삭제</button>
+        <button class="checked-btn">완료</button>
+        
+      </div>
+      <table class="todo-table">
+          <tr class="todo-header">
+            <th><input class="all-check" type="checkbox"></th>
+            <th>중요도</th>
+            <th>완료됨</th>
+            <th>할 일</th>
+          </tr>
+      </table>
```

Comment:
```suggestion
<table class="todo-table">
  <thead class="todo-header">…</thead>
  <tbody class="todo-body"></tbody>
</table>
```

이렇게 시멘틱하게 바꾸어주고 -> 자바스크립트로는 header 를 매번 cloneNode 하지 않고 body만 갱신 시킬 수 있어서 시멘틱 요소도 가져가고 로직도 아주.. 조금 간결해질 거 같아요 ㅎㅎ 물론 지금도 좋습니다 !!!